.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: false
 services:
   - docker
 script:
-  - bash ./build-docker.sh
+  - bash ./tools/build-docker.sh
```

Comment:
Could you please update the invocation of this script in the Building the Docker Images Locally section of the readme?

Reply:
Sure

Reply:
Added